lib/server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const Hapi = require('@hapi/hapi');
+
+const server = Hapi.server({
+    port: 3000,
+    host: 'localhost'
+});
+
+server.route(require('./routes'));
+
+exports.init = async () => {
+
+    await server.initialize();
+    return server;
+};
+
+exports.start = async () => {
+
+    await server.start();
+    console.log(`Server running at: ${server.info.uri}`);
+    return server;
+};
+
+process.on('unhandledRejection', (err) => {
+
+    console.log(err);
+    process.exit(1);
+});
```

Comment:
This was just moving this code into a another file? Is that required for testing?